bitnami/mariadb-galera/values.yaml

```diff
@@ -319,7 +319,7 @@ mariadbConfiguration: |-
   tmpdir=/opt/bitnami/mariadb/tmp
   socket=/opt/bitnami/mariadb/tmp/mysql.sock
   pid_file=/opt/bitnami/mariadb/tmp/mysqld.pid
-  bind_address=0.0.0.0
+  bind_address=::
```

Comment:
What would happen if IPv6 is not available?

Reply:
I'm testing in minikube which does not support ipv6.
here's proof:

Reply:
@alemorcuq

Reply:
If the address is `::`, the server accepts TCP/IP connections on all server host IPv4 and IPv6 interfaces. Use this address to permit both IPv4 and IPv6 connections on all server interfaces.